### DISCUSSION TITLE
Pane inside a pane at a specified location

### DISCUSSION THREAD

**Discussion by okay1204** [stefvanschie/IF#225] [Q&A]
Is it possible to have a pane inside a pane? I know MasonryPane exists, however you can't really control the positioning of the panes that well with this. I want to be able to choose an x and y coordinate for each pane inside the pane.

**Comment by stefvanschie** [comment]
For guis/inventory components, you can place multiple panes inside the gui by calling `addPane(Pane)` for each pane you want to add. For `PaginatedPane` you can add more panes inside of it by calling `addPane(int, Pane)` for each pane, possibly also on the same page.

**Comment by okay1204** [comment]
Ah, I didn't realize you could have multiple panes on the same page. Thanks
